cmd/spartakus/volunteer.go

```diff
@@ -42,6 +43,7 @@ func (_ volunteerSubProgram) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&volunteerConfig.database, "database",
 		"https://spartakus.k8s.io", "Send reports to this database; use --print-databases for a list of options")
 	fs.BoolVar(&volunteerConfig.printDatabases, "print-databases", false, "Print database options and exit")
+	fs.StringVar(&volunteerConfig.extensionsPath, "extensions", "", "Path to a file of additional metrics to report; leave unset to report no additional metrics")
```

Comment:
Is it sufficient to have a single extensions file? Given that this will read async to any writer, the writer will have to do something like write to a temp file, close it, then rename() it to the final extensions filename.  Do we ever need more than one such writer?